src/commands/edge/drivers/logcat.ts

```diff
@@ -229,8 +245,12 @@ export default class LogCatCommand extends SseCommand<typeof LogCatCommand.flags
 			}
 		}
 
+		const minLogLevel = await this.getMinLogLevel()
+
 		this.source.onmessage = (event: MessageEvent<LiveLogMessage>) => {
-			logEvent(event, liveLogMessageFormatter)
+			if(event.data.log_level >= minLogLevel) {
```

Comment:
Could you please enable [keyword-spacing](https://eslint.org/docs/latest/rules/keyword-spacing#rule-details) with this change?

Reply:
Done.